lib/evil_query.rb

```diff
@@ -86,7 +86,7 @@ def community_spotlight_posts!
     end
 
     def ids!(query)
-      fetch_ids!(query.for_a_stream(@max_time, @order), 'posts.id')
+      fetch_ids!(query.for_a_ranged_stream(@max_time - 30, @max_time, @order), 'posts.id')
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.